web/src/features/fireWeather/components/TimeOfInterestPicker.tsx

```diff
@@ -37,7 +36,7 @@ const TimeOfInterestPicker = (props: Props) => {
       onChange={e => {
         const value = e.currentTarget.value
         if (value) {
-          props.onChange(new Date(value))
+          props.onChange(`${value}:00-08:00`) // Append seconds and timezone at the end
```

Comment:
Is this going to work in all locales?

Reply:
is there a nice straightforward way to determine a user's timezone/T offset based on their OS or browser or something like that?

Reply:
At this point yes, because that value doesn't contain the timezone information. But this is actually a valid concern when initializing the time of interest. I need to make a change in `getTimeOfInterestFromUrl`

Reply:
@andrea-williams there is https://stackoverflow.com/questions/1091372/getting-the-clients-time-zone-and-offset-in-javascript
In this case tho we are fixating the timezone to PST